src/react-native/11-navigation/navigation.md

```diff
@@ -236,7 +236,7 @@ Similar to the `Tab.Screen` component, we can customize the behavior of our `Sta
 
 ### Setup 2
 
-✏️ Run:
+✏️ Install the new dependency:
```

Comment:
`dependency` -> `dependencies`

Reply:
damn. I thought i got them all

Reply:
Some said "Install X and Y and Z" and then the next line said "`npm install X Y Z`" so that just felt really redundant. I changed them to be more informative, but then needed to update the rest to say more than `Run`.